### PR TITLE
short array updated in code examples

### PR DESCRIPTION
**Pull request by milindsingh**
## Purpose of this pull request

This pull request (PR) update `array()` to `[]` in multiple code examples.

## Affected DevDocs pages

-  https://devdocs.magento.com/guides/v2.3/get-started/soap/soap-web-api-calls.html
-  https://devdocs.magento.com/cloud/env/variables-global.html
-  https://devdocs.magento.com/guides/v2.3/coding-standards/code-standard-demarcation.html
-  https://devdocs.magento.com/guides/v2.3/config-guide/mq/manage-message-queues.html
-  https://devdocs.magento.com/guides/v2.3/extension-dev-guide/test/test_object-mgr.html

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: src/cloud/env/variables-global.md [hunk 1]
```diff
@@ -50,11 +50,11 @@ stage:
 The `SCD_ON_DEMAND` variable skips the SCD in both phases (build and deploy), clears the `pub/static` and `var/view_preprocessed` folders, and writes the following to the `app/etc/env.php` file:
 
 ```php?start_inline=1
-return array(
+return [
```
Comment (hguthrie) [inline src/cloud/env/variables-global.md:53]: When SCD_ON_DEMAND is set to true, the `env.php` reflects the following change in the remote Cloud environment:
Screen Shot 2020-07-31 at 09 42 03

Please revert changes to the Cloud topics.

**Comment by dshevtsov** [comment]
@lenaorobei please confirm whether this complies with Magento coding standards.

File: src/cloud/env/variables-global.md (hunk 1 above)
Comment (milindsingh) [reply src/cloud/env/variables-global.md:53]: I will, but didnt understand why use the old array syntax in example.

Comment (hguthrie) [reply src/cloud/env/variables-global.md:53]: In this example, we show the outcome of adding the environment variable to the configuration file. After deploying the new configuration, this is what Cloud automatically adds to the `env.php` file. We don't have control over that and we are not advocating manual changes to the file in this topic.

Comment (milindsingh) [reply src/cloud/env/variables-global.md]: It's strange. I'll check later whats happening in the cloud. 
I have removed the change for now.

Comment (milindsingh) [reply src/cloud/env/variables-global.md]: @hguthrie kindly approve.

**Comment by meker12** [comment]
running tests

**Pull request merged**